templates/deployments.yaml

```diff
@@ -780,56 +780,56 @@ spec:
 {{ toYaml . | indent 8 }}
     {{- end }}
 
+{{- range .Values.st2.packs.sensors }}
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}-st2sensorcontainer{{ template "enterpriseSuffix" . }}
+  name: {{ $.Release.Name }}-st2sensorcontainer-{{ .name }}{{ template "enterpriseSuffix" $ }}
```

Comment:
Just thinking out loud: it could be a matter of time when `name` exceeds the limit of number of allowed characters.
But we have no solutions here, except of cutting/trimming it. A task for some future.

Reply:
The current container names are about 75 characters long including the resource prefix `/api/v1/pods`. The maximum length of a Kubernetes resource is 253 characters. Unless I'm missing something obvious, I do not think we'll exceed any limits.

Reply:
See:
https://github.com/helm/charts/blob/d712e0cfd7d1426b6adb59473530fcc84bacbaf2/stable/rabbitmq-ha/templates/_helpers.tpl#L11-L12
https://github.com/helm/helm/issues/2022